dts/bindings/regulator/regulator-fixed.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019-2020 Peter Bigot Consulting, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO-controlled regulators
+
+include: regulator.yaml
+
+# NOTE: The driver supports "regulator-fixed-sync" as a specializing
+# variant when it's known that the GPIO state change operations are
+# isr-ok and not sleep, and both startup and off-on delays are zero.
+# This bypasses the asynchronous onoff manager which optimizes both time
+# and memory use.
+#
+# To enable this use both:
+#    compatible = "regulator-fixed-sync", "regulator-fixed";
+#
+
+compatible: "regulator-fixed"
+
+properties:
+  label:
+    required: true
+
+  regulator-name:
+    type: string
+    required: true
+    description: Descriptive name for regulator outputs.
+
+  enable-gpios:
```

Comment:
Looks like upstream just calls this `gpio` for this property.

Reply:
Yes, that's true.  upstream uses `gpio`, `enable-active-high`, and `gpio-open-drain`.  This uses `enable-gpio` to handle all three of those.

I though we'd discussed this and you agreed we shouldn't attempt to support the extra properties when proper use of a GPIO selector makes them unnecessary.  Given that, I think we should use a different name, because `gpio` would suggest the flags are ignored.

Reply:
If we need to name this `zephyr,regulator-fixed` to avoid confusion I'm fine with that.

Reply:
I think we can leave things as is, but can we add a comment in the binding to explain how/why `enable-gpios` differs from `gpio`?

Reply:
@galak Rebased (since this hasn't been tested directly on master since August), and then [updated the text](https://github.com/zephyrproject-rtos/zephyr/compare/577031aaef1a3f6c0f0c54d5611e6093b5c53884..1056280ca4c5348d839fd6ce39816a8c0877376e).

Reply:
I went with documentation text rather than comment, because if we're going to generate human-readable documentation for these things the comments are less visible than when we expect people to look at the binding file directly.